app/editor/src/features/admin/topics/TopicFormSmall.tsx

```diff
@@ -0,0 +1,79 @@
+import { FormikForm } from 'components/formik';
+import React from 'react';
+import { toast } from 'react-toastify';
+import {
+  Button,
+  ButtonVariant,
+  FieldSize,
+  FormikText,
+  ITopicModel,
+  Row,
+  ToggleGroup,
+  TopicTypeName,
+} from 'tno-core';
+
+import { defaultTopic } from './constants';
+import * as styled from './styled';
+
+interface ITopicFormSmallProps {
+  onAddOrUpdate: (value: ITopicModel) => void;
+}
+
+/** The page used to view and edit tags in the administrative section. */
+export const TopicFormSmall: React.FC<ITopicFormSmallProps> = ({ onAddOrUpdate }) => {
+  const handleSubmit = async (values: ITopicModel) => {
+    try {
+      onAddOrUpdate?.(values);
+    } catch {
+      toast.error(`Error saving topic ${values.name}.`);
```

Comment:
This will be a double toast as all our hooks inherit error handling that makes a toast error already.